cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+    "    \n",
+    "If working with images, `image_paths` should be a string of absolute or relative paths to the where each index corresponds the example for that row of `multiannotator_labels`.\n",
+    "\n",
+    "If working with other data, `image_paths` should be a string of examples corresponding to the row of `multiannotator_labels`.\n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "66bafbf8",
+   "metadata": {},
+   "source": [
+    "Before training our machine learning model, we must first obtain the consensus labels from the annotators that labeled the data. The simplest way to obtain an initial set of consensus labels is to select it using majority vote."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5a953e3a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "consensus_labels = get_majority_vote_label(multiannotator_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c114734",
+   "metadata": {},
+   "source": [
+    "## 3. Train model and use cleanlab to get better consensus labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e31453ef",
+   "metadata": {},
+   "source": [
+    "Next, we will train our model on the consensus labels obtained using majority vote to compute out-of-sample predicted probabilities. We will then use these `pred_probs` to generate more informed `consensus_labels` using Cleanlab's [CROWDLAB](link to something) algorithm. We them use these `consensus_labels` to train a better model that generates more accurate `pred_probs`. This process itterates until the `consensus_labels` have no more improvement.\n",
+    "\n",
+    "`train_model()` trains a `resnet18` image model using cross validation to get out-of-sample predicted probabilities on the whole dataset. The function can be replaced with a custom training algorithm."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "370c66ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_type = \"resnet18\" # You can also try with \"swin_base_patch4_window7_224\"\n",
+    "\n",
+    "# Load model arguments\n",
+    "train_args = {\n",
+    "    \"num_cv_folds\": 5, \n",
+    "    \"verbose\": 1, \n",
+    "    \"epochs\": 1, \n",
+    "    \"holdout_frac\": 0.2, \n",
+    "    \"time_limit\": 21600, \n",
+    "    \"random_state\": 123\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7c27da16",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Loop through and retrain model on better consensus labels, save results\n",
+    "indices_changed = set()\n",
+    "seen_consensus_labels = list()\n",
```

Comment:
would this work as `set()` instead of `list()`?  I don't know

Reply:
I kept it as a list because I track the number of changes from the previous method itteration of consensus labels. I.e. `seen_consensus_labels[-1]` but that metric is not great and I can remove it / make this a set

Reply:
Up to you what you think is best based on your experience here, I really don't know